central/complianceoperator/v2/report/manager/watcher/scanconfigwatcher.go

```diff
@@ -49,18 +51,19 @@ type scanConfigWatcherImpl struct {
 	scanC   chan *ScanWatcherResults
```

Comment:
The name here may suggest that this channel holds scans, but in fact it holds scan results. I was worried that we push scan results to two different channels (here and in `scanWatcherImpl`), but that is not the case, just name is similar.

Reply:
I was wrong, this channel does not hold `ComplianceOperatorCheckResultV2` but `ScanWatcherResults`. This requires careful navigation through code, as both things are named results.